frontend/src/components/transactions/TransactionDetails.tsx

```diff
@@ -103,24 +105,22 @@ export default class extends React.Component<Props, State> {
             <CardCell
               title="Total Gas Cost"
               imgLink="/static/images/icon-m-size.svg"
-              text={<Balance amount={totalFee.toString()} />}
+              text={<Balance amount={transactionFee.toString()} />}
```

Comment:
One day we should consider using BN type instead of strings throughout our code base.

Reply:
nearlib uses string to tackle with amount, so maybe  give issue in nearlib

Reply:
Yes, that would be also helpful. Especially once we change RPC to encode almost (?) all numbers as strings (we mostly use u64 values and those do not fit into JS number type, so to avoid loss in precision, we should encode as strings and deserialize those into BN for nicer API on near-api-js [fka nearlib] side).